tests/integration/test_batch_notification_processor_updates_message_queue.py

```diff
@@ -54,10 +59,21 @@ def test_batch_notification_processor_payload(batch_id, recipient_data, helpers)
 
     assert adapter.called
     assert adapter.call_count == 1
+
+    try:
+        validate(instance=adapter.last_request.json(), schema=request_schema)
+    except ValidationError as e:
+        return False, e.message
```

Comment:
@TomSmithCoding Does this return statement exit the test here? Would it be better to use an assertion so that the test can continue and pick up any other failures?